test_scripts/Capabilities/UpdateVideoStreamingCapabilities/GetSystemCapability/004_GetSystemCapability_after_OnSystemCapabilityUpdated_same_app.lua

```diff
@@ -0,0 +1,48 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description:  SDL successfully transfers received from HMI on startup videoStreamingCapabilities
```

Comment:
Update the description(the idea of script to check caching of videoStreamingCapabilities after receiving OnSystemCapabilityUpdated)